server/src/main/java/org/elasticsearch/search/vectors/KnnSearchBuilder.java

```diff
@@ -151,6 +204,10 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeVInt(k);
         out.writeVInt(numCands);
         out.writeFloatArray(queryVector);
+        out.writeVInt(filterQueries.size());
```

Comment:
Should we write filterQueries the same way we read them:
`out.writeNamedWriteableList(filterQueries);`?

Reply:
Good point, not sure why I did this manually.